src/cli.js

```diff
@@ -420,39 +418,21 @@ async function build() {
     let resolvedConfig = resolveConfigInternal(config)
 
     if (args['--purge']) {
-      resolvedConfig.purge = {
-        enabled: true,
-        content: args['--purge'].split(/(?<!{[^}]+),/),
+      log.warn(['The `--purge` flag has been deprecated.', 'Please use `--content` instead.'])
+      if (!args['--content']) {
+        args['--content'] = ['--purge']
```

Comment:
This should probably be `args['--content'] = args['--purge']`

The current version will just assign an array with `--purge` as an element

Reply:
Whoops, just realised I commented on a merged branch 🤦‍♂️

Reply:
No worries, and thanks for notifying! I've fixed it here: https://github.com/tailwindlabs/tailwindcss/pull/5646